plugins/scaffolder-backend/src/scaffolder/actions/builtin/filesystem/delete.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTemplateAction } from '../../createTemplateAction';
+import { InputError } from '@backstage/errors';
+import { resolveSafeChildPath } from '@backstage/backend-common';
+import fs from 'fs-extra';
+
+export const createFilesystemDeleteAction = () => {
+  return createTemplateAction<{ files: string[] }>({
+    id: 'fs:delete',
```

Comment:
You can also pass in a `description` to this object to describe what the action does that will make it look nice when you visit `/create/actions`.

Reply:
✅ updated in [`3d24e8a` (#6340)](https://github.com/backstage/backstage/pull/6340/commits/3d24e8aecf92d41a1d4916aaa4f69180e38683bc#diff-7ae77f1bed507381de74464ea82a9ce01be3ee2997e11e180aa8dab9680f9359R49)